### DISCUSSION TITLE
Sending proof

### DISCUSSION THREAD

**Discussion by GUIPETAV** [community/community#133993] [GitHub Education]
### Hi everyone,

Hi everyone, I need to attach my proof that would give me the right to use GitHub Education, but I have not received any alternative support. I am currently a teacher and would like to explore the tool with my students.

**Comment by 0x-Apollyon** [comment]
Did you submit the application with a school letterhead or any other document which proves that you are a teacher ?
Also I suggest waiting because it takes Github 2-3 days to process your request
If it still doesnt work you can contact github at https://support.github.com/

**Comment by Faiaz3k** [comment]
let me suggest some staple
1 Gather Your Proof
2 go to https://education.github.com/
3 Create a GitHub Account
4 Apply for Teacher Toolbox
5 Submit Your Application

**Comment by juanpflores** [comment]
Ola!
Mind checking your email in the next three days? I've taken a look at your request 😉 

Best
